src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -266,39 +266,39 @@ SELECT shards_colocated(1300000, 1300020);
 (1 row)
 
 -- check co-located table list
-SELECT UNNEST(get_colocated_table_array('table1_group1'))::regclass;
+SELECT UNNEST(get_colocated_table_array('table1_group1'))::regclass ORDER BY 1;
```

Comment:
I have a similar commit (517ac89e312b9e5c7569e3edceebd88883c0716b) in `fix_pg_11`, but I guess I'll merge this and remove my commit, since they're almost identical.